caliban/docker.py

```diff
@@ -95,10 +95,10 @@ def __str__(self) -> str:
 def apt_install(*packages: str) -> str:
   """Returns a command that will install the supplied list of packages without
   requiring confirmation or any user interaction.
-
   """
   package_str = ' '.join(packages)
-  return "apt-get install --yes --no-install-recommends {}".format(package_str)
+  no_prompt = "DEBIAN_FRONTEND=noninteractive"
```

Comment:
I've hit this before, not trivial to find the magic incantation of DEBIAN_FRONTEND!